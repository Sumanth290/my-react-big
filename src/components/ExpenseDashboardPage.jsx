import React from "react";
import ExpenseList from "./ExpenseList.jsx";
import ExpenseListFilters from "./ExpenseListFilters.jsx";
import ExpenseSummary from "./ExpenseSummary.jsx";


const ExpenseDashboardPage = () => (
    <div>
        <h2>This is "Expense Dashboard Home Page"</h2>
        <ExpenseSummary />
        <ExpenseListFilters />
        <ExpenseList />
    </div>
);

export default ExpenseDashboardPage;