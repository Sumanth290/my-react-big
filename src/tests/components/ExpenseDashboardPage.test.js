import React from "react";
import {shallow} from "enzyme";
import ExpenseDashboardPage from "../../components/ExpenseDashboardPage";
import toJSON from "enzyme-to-json";


test("Should render ExpenseDashboardPage",()=>{
    const wrapper = shallow(<ExpenseDashboardPage />);
    expect(toJSON(wrapper)).toMatchSnapshot();
});