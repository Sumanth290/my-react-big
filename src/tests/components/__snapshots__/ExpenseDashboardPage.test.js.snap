// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ExpenseDashboardPage 1`] = `
<div>
  <h2>
    This is "Expense Dashboard Home Page"
  </h2>
  <Connect(ExpenseSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
